JASP-Engine/JASP/R/multinomialtest.R

```diff
@@ -643,7 +643,13 @@ MultinomialTest <- function (dataset = NULL, options, perform = "run",
     .quitAnalysis(paste0(variable, "variable contains negative values"))
   }
 
-  if (!all(counts == round(counts))) {
+  # only applies for observed counts, expected counts can be proportions
+  if (!expectedCounts & !all(counts == round(counts))) {
```

Comment:
I'm guessing you meant to use  `&&` here?

Reply:
Corrected in latest commit.